mosviz/loaders/loader_selection.py

```diff
@@ -203,6 +203,14 @@ def _validation_checks(self, *args, **kwargs):
                 self.data._load_log.path.split(os.sep)[:-1])
 
             for filename in filenames:
+                if filename.count("[") > 1 or filename.count("]") > 1:
```

Comment:
I'm not sure this kind of validation is all that useful or necessary. It wouldn't catch any of the following errors: `[filename.fits]`
`filename.fits]0[`
etc...

So it might be best just to drop this error checking. See comment about the use of regex below.

Reply:
Very good point! I think the validation itself will fail indicating that the file name is bad. Removing these lines.